_layouts/raw.html

```diff
@@ -28,7 +28,54 @@
 <script src="//cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
 </head>
 <body class="mdl-layout mdl-js-layout mdl-layout--fixed-header mdl-layout--overlay-drawer-button">
-<header class=mdl-layout__header>
+    <div class="mdl-layout mdl-js-layout mdl-layout--fixed-header">
+  <header class="mdl-layout__header">
+    <div class="mdl-layout__header-row">
+      <!-- Title -->
+      <span class="mdl-layout-title"><a href="{{site.baseurl}}/" class="mdl-layout-title mdl-navigation__link">IRkernel Documentation</a></span>
+      <!-- Add spacer, to align navigation to the right -->
+      <div class="mdl-layout-spacer"></div>
+      <!-- Navigation. We hide it in small screens. -->
+      <nav class="mdl-navigation mdl-layout--large-screen-only">
+        <a class="mdl-navigation__link" href={{site.baseurl}}/requirements/><i class="material-icons nav-icon">assessment</i>Requirements</a>
+        <a class="mdl-navigation__link" href={{site.baseurl}}/installation/><i class="material-icons nav-icon">file_download</i>Installation</a>
+        <a class="mdl-navigation__link" href={{site.baseurl}}/running/><i class="material-icons nav-icon">launch</i>Running</a>
+        <a class="mdl-navigation__link" href={{site.baseurl}}/faq/><i class="material-icons nav-icon">question_answer</i>FAQ</a>
+      </nav>
+    </div>
+  </header>
+  <div class="mdl-layout__drawer">
+    <span class="mdl-layout-title">IRkernel Docs</span>
+    <nav class="mdl-navigation">
+        <a class="mdl-navigation__link" href={{site.baseurl}}/requirements/><i class="material-icons mobile-nav-icon">assessment</i>Requirements</a>
+        <a class="mdl-navigation__link" href={{site.baseurl}}/installation/><i class="material-icons mobile-nav-icon">file_download</i>Installation</a>
+        <a class="mdl-navigation__link" href={{site.baseurl}}/running/><i class="material-icons mobile-nav-icon">launch</i>Running</a>
+        <a class="mdl-navigation__link" href={{site.baseurl}}/faq/><i class="material-icons mobile-nav-icon">question_answer</i>FAQ</a>
+        <span class="seperator"></span>
+        <a class="mdl-navigation__link" href="https://github.com/IRkernel"><i class="mega-octicon octicon-mark-github mobile-nav-icon"></i>IRkernel on GitHub</a>
+        <a class="mdl-navigation__link" href="https://github.com/IRkernel/IRdisplay"><i class="mega-octicon octicon-mark-github mobile-nav-icon"></i>IRDisplay on GitHub</a>
+        <a class="mdl-navigation__link" href="https://github.com/IRkernel/repr"><i class="mega-octicon octicon-mark-github mobile-nav-icon"></i>repr on GitHub</a>
```

Comment:
I'm not particularly fond of hamburger menus for desktop UIs, though I accept they're a useful convention on smaller phone screens. Maybe the Github links could go at the bottom of the homepage? Design's not my thing, though, so if you think the hamburger menu is the best option even on large screens, I'll bow to your expertise. ;-)